### PR TITLE
Fix related dim bug(when a_outer = 0)

### PR DESCRIPTION
**Pull request by haozech**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ```
                         ___                    ___          ___     
-                       /\__\                  /\  \        /\  \    ______________
+                       /\__\                  /\  \        /\  \
```
Comment (wenming2014) [inline README.md:3]: ?

File: cinn/poly/isl_utils.cc [hunk 2]
```diff
@@ -358,19 +359,29 @@ isl::map RemoveAxiesByOutputNames(const isl::map &x, const std::vector<std::stri
 
 std::vector<std::string> GetRelatedOutputAxies(const isl::map &x, const std::vector<std::string> &dim_in_names) {
   std::string map_str = isl_map_to_str(x.get());
-  isl::ctx this_ctx   = x.ctx();
+  LOG(INFO) << "GetRelatedOutputAxies map_str is : " << map_str;
```
Comment (wenming2014) [inline cinn/poly/isl_utils.cc]: VLOG,下同

File: README.md (hunk 1 above)
Comment (haozech) [reply README.md:3]: 修正ASCII字符画 见此issue https://github.com/PaddlePaddle/CINN/issues/385

File: cinn/poly/isl_utils.cc (hunk 2 above)
Comment (haozech) [reply cinn/poly/isl_utils.cc]: 已修改

**Review by wenming2014 — APPROVED** [review]

**Pull request merged**